_assets/scss/elements/_typography.scss

```diff
@@ -88,6 +99,7 @@
 .h5--med { @include h5--med; }
 .h5--med { @include h5--med; }
 
+// $fix – box works on Chrome but not on safari
```

Comment:
Will try to see if it can be fixed by keeping the :after structure. Probably it's just missing :active along with :focus